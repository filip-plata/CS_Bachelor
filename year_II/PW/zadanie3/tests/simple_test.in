2 1 2
1 x[0] = ((-1000) + 50)
2 x[1] 5
