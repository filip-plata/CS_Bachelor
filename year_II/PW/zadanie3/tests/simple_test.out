1 P
2 P -950
